### ISSUE TITLE
Expand output cells to use available vertical space

### ISSUE THREAD

**Issue by nlgranger** [damianavila/RISE#639]
Slides are centered vertically based on the initial content height.
As output grows, the slide does not remain centered, instead the content just expands downward beyond the viewport.
The slide ends up with a weird blank space at the top, and more often than not the scroll bar gets confused.

![image](https://user-images.githubusercontent.com/3764009/232222206-68263d39-02c0-4081-8d95-4ba4cf79470c.png)
![image](https://user-images.githubusercontent.com/3764009/232222893-bafe1749-f62e-441d-97a2-61aa22a09039.png)

I'd suggest the following behaviors instead:
- Empty output cells use all available vertical space by default, and expand beyond if necessary causing a scrollbar to appear (on the slide or the output cell only?)
- Provide a way to manually set height in cell metadata
- Provide an option to align the slides to the top vertically, so there is no gap above the content to begin with.

**Comment by nlgranger** [comment]
Found a workaround.

**Comment by nlgranger** [comment]
Rephrased the issue.

**Comment by parmentelat** [comment]
have you tried to set `center = false` in your rise config ? 
this is handed over to reveal.js, that should stop doing the vertical centering; which iiuc is the point of your last bullet

**Comment by nlgranger** [comment]
Indeed, but the scrollbar doesn't show up when needed as output grows.

**Comment by parmentelat** [comment]
and have you also said `scroll=true` as per
https://rise.readthedocs.io/en/stable/customize.html?highlight=scroll#enable-a-right-scroll-bar <https://rise.readthedocs.io/en/stable/customize.html?highlight=scroll#enable-a-right-scroll-bar>

if so, I’m afraid there is not much more I can think of..
primarily only a fraction of the settings that you give to rise are for rise itself, the rest - like, precisely, center and scroll, are passed along to reveal.js; maybe look in that documentation if you’re still stuck
